examples/integration_with_error.py

```diff
@@ -123,13 +110,12 @@
         manual_bounds,
         ".",
         capsize=6,
-        label=r"bounds $%4g\rightarrow%4g$" % bounds,
+        label="%s (programmed σ)\n$%4g\\rightarrow%4g$" % ((thislabel,)+bounds),
```

Comment:
more informative labels for both times through the loop

Reply:
did you just copy and paste the sigma?

Reply:
`<ctrl-K>s*`